Scripts/Utils.ps1

```diff
@@ -0,0 +1,39 @@
+﻿function CleanTempFolder()
+{
+    if (Test-Path $env:temp)
+    {
+        Write-Host 'Deleting temp folder'
+        rmdir $env:temp -Recurse -ErrorAction SilentlyContinue
```

Comment:
Probably! The scripts are CI biased. I will revisit this comment for the next PR on this.